layout/new.go

```diff
@@ -58,6 +63,67 @@ func NewImage(path string, ops ...imgutil.ImageOption) (*Image, error) {
 	}, nil
 }
 
+// NewIndex will return an OCI ImageIndex saved on disk using OCI media Types. It can be modified and saved to a registry
+func NewIndex(repoName, path string, ops ...imgutil.IndexOption) (idx *ImageIndex, err error) {
+	var mfest *v1.IndexManifest
+	var idxOps = &imgutil.IndexOptions{}
+	for _, op := range ops {
+		if err = op(idxOps); err != nil {
+			return idx, err
+		}
+	}
+
+	if err = validateRepoName(repoName, idxOps); err != nil {
+		return idx, err
+	}
+
+	if idxOps.BaseIndex == nil && idxOps.BaseImageIndexRepoName != "" {
+		idxOps.BaseIndex, err = newImageIndexFromPath(idxOps.BaseImageIndexRepoName)
+		if err != nil {
+			return idx, err
+		}
+
+		if idxOps.BaseIndex != nil {
+			// TODO Do we need to do this?
```

Comment:
Ehh probably not

Reply:
I think we should removes it, we try to load an index from path and according to the option `FromBaseImageIndex`, **If the index is not found, it does nothing** and we are throwing an error

Reply:
I deleted this code block in https://github.com/husni-faiz/imgutil/pull/12, but I am not sure if I fixed

> If the index is not found, it does nothing and we are throwing an error